src/controllers/__tests__/events.test.js

```diff
@@ -150,4 +150,26 @@ describe("Testing events for routes doesn't require auth controls", () => {
         return events;
       });
   });
+
+  it('PATCH /api/events/:id should update an existing event', (done) => {
+    request(app)
+      .patch(`/api/events/1/`)
+      .send(events[0])
+      .expect('Content-Type', /json/)
+      .expect(200, (err, res) => {
+        if (err) {
+          done();
+          return err;
+        }
+        if (!res.body) {
+          expect(res.statusCode).toBe(404);
+          done();
+          return res.json('No event found');
+        }
```

Comment:
Please, check this thoroughly, almost all of the code here is wrong. `req.body` will always be truthy. and `res.json` doesn't work here